app/javascript/dashboard/routes/dashboard/conversation/ContactPanel.vue

```diff
@@ -4,118 +4,158 @@
       <i class="ion-chevron-right" />
     </span>
     <contact-info :contact="contact" :channel-type="channelType" />
-    <div class="conversation--actions">
-      <accordion-item
-        :title="$t('CONVERSATION_SIDEBAR.ACCORDION.CONVERSATION_ACTIONS')"
-        :is-open="isContactSidebarItemOpen('is_conv_actions_open')"
-        @click="value => toggleSidebarUIState('is_conv_actions_open', value)"
-      >
-        <div>
-          <div class="multiselect-wrap--small">
-            <contact-details-item
-              compact
-              :title="$t('CONVERSATION_SIDEBAR.ASSIGNEE_LABEL')"
-            >
-              <template v-slot:button>
-                <woot-button
-                  v-if="showSelfAssign"
-                  icon="ion-arrow-right-c"
-                  variant="link"
-                  size="small"
-                  @click="onSelfAssign"
-                >
-                  {{ $t('CONVERSATION_SIDEBAR.SELF_ASSIGN') }}
-                </woot-button>
-              </template>
-            </contact-details-item>
-            <multiselect-dropdown
-              :options="agentsList"
-              :selected-item="assignedAgent"
-              :multiselector-title="$t('AGENT_MGMT.MULTI_SELECTOR.TITLE.AGENT')"
-              :multiselector-placeholder="
-                $t('AGENT_MGMT.MULTI_SELECTOR.PLACEHOLDER')
-              "
-              :no-search-result="
-                $t('AGENT_MGMT.MULTI_SELECTOR.SEARCH.NO_RESULTS.AGENT')
+    <draggable
```

Comment:
Please consider making this a separate component, this component has grown beyond 300 lines.